pyproject.toml

```diff
@@ -6,25 +6,11 @@ requires = [
     "Cython>=0.28.5",
 
     # PyPy needs numpy >= 1.14.0
-    # platform_python_implementation!='CPython' not needed >= Python 3.7 which is numpy 1.14.5
-    "numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation=='CPython'",
-    "numpy==1.14.0; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation!='CPython'",
+    "numpy==1.14.0; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation=='PyPy'",
 
-    # AIX needs numpy >= 1.16.0
-    # platform_system!='AIX' not needed >= Python 3.8 which is numpy 1.17.3
-    "numpy==1.16.0; python_version=='3.6' and platform_machine!='aarch64' and platform_system=='AIX'",
-    "numpy==1.16.0; python_version=='3.7' and platform_machine!='aarch64' and platform_system=='AIX'",
-
-    # ARM needs numpy >= 1.19.0
-    # platform_machine!='aarch64' not needed >= Python 3.9 which is numpy 1.19.3
-    "numpy==1.19.0; python_version=='3.6' and platform_machine=='aarch64'",
-    "numpy==1.19.0; python_version=='3.7' and platform_machine=='aarch64'",
-    "numpy==1.19.0; python_version=='3.8' and platform_machine=='aarch64'",
-
-    # default numpy requirements
-    "numpy==1.14.5; python_version=='3.7' and platform_machine!='aarch64' and platform_system!='AIX'",
-    "numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64'",
-    "numpy==1.19.3; python_version=='3.9'",
+    # use oldest-supported-numpy where possible (not for pypy on Python 3.6)
```

Comment:
Can we add a comment that links to https://github.com/scipy/oldest-supported-numpy/blob/master/setup.cfg so we can easily refer to it as needed?

Reply:
Sound good ot me!